server/src/services/UserServices.js

```diff
@@ -86,6 +86,32 @@ const UserServices = {
     });
   },
 
+  async checkScuntToken(existingUser) {
+    if (!existingUser.scuntToken && existingUser.isRegistered == true && existingUser.scunt) {
+      return false;
+    }
+    return true;
+  },
+
+  async getScuntToken(email) {
```

Comment:
It looks like this is adding a scunt token to a user. If that is the case, it should be renamed to reflect the fact that it is adding, not getting, a token for a user